docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2706,6 +2718,45 @@ definitions:
       - event_type
       - partitions
 
+  SubscriptionEventTypeLightStats:
+      type: object
+      description: light statistics of one event-type within a context of subscription
+      properties:
+        event_type:
+          type: string
+          description: event-type name
+        partitions:
+          type: array
+          description: statistics of partitions of this event-type
+          items:
+            type: object
+            description: statistics of partition within a subscription context
+            properties:
+              partition:
```

Comment:
there's no description for `partition` field

Reply:
f650f43